src/config.py

```diff
@@ -39,6 +39,32 @@ def sync_password(self) -> Optional[str]:
         """Returns charm-set sync_password for server-server auth between brokers."""
         return self.charm.get_secret(scope="app", key="sync-password")
 
+    @property
+    def internal_port(self) -> int:
+        """Returns the port used for INTERNAL listener."""
+        return 9093 if self.charm.tls.enabled else 9092
+
+    @property
+    def external_port(self) -> int:
+        """Returns the port used for EXTERNAL listener."""
+        return 19092
```

Comment:
Should this one be in a similar fashion to the internal port?
```python
return 19093 if self.charm.tls.enabled else 19092
```

Reply:
We can use the existing `literals.py` for that.

Reply:
And @zmraul to answer your question, yes, `19093` for EXTERNAL + SSL, `19092` for EXTERNAL + PLAINTEXT